server/bin/seeds.js

```diff
@@ -1,43 +1,139 @@
+    } catch (err) {
+        console.log(`Could not populate documents for ${name}: ${err.name} - ${err._message}`, err)
+    }
+}
+
+async function createDBEntries() {
+    await submitDocuments('users', User, Array.from({ length: 10 }).map(e => {
+        return {
+            username: faker.name.findName(),
+            password: bcrypt.hashSync(faker.internet.password(), bcrypt.genSaltSync(bcryptSalt)),
+            email: faker.internet.email()
+        }
+    }))
+
+    await submitDocuments('posts', Post, Array.from({ length: 100 }).map((e, i) => {
+        return {
+            title: faker.lorem.sentence(),
+            content: faker.lorem.paragraph(),
+            author: databaseEntries.users[i % databaseEntries.users.length]._id
+        }
+    }))
+
+    await submitDocuments('comments', Comment, Array.from({ length: 1000 }).map((e, i) => {
+        return {
+            content: faker.lorem.paragraph(),
+            post: databaseEntries.posts[i % databaseEntries.posts.length]._id,
+            author: databaseEntries.users[i % databaseEntries.users.length]._id
+        }
+    }))
+
+    await submitDocuments('categories', Category, require('./categories.json'))
+
+    await submitDocuments('games', Game, Array.from({ length: 100 }).map(e => {
+        return {
+            name: faker.commerce.productName(),
+            description: faker.lorem.sentence(),
+            price: faker.commerce.price(1, 1000, 2, ''),
+            image: faker.image.abstract(),
+            year_published: Number(faker.date.between('1990-01-01', new Date().toISOString().slice(0, 10)).toISOString().slice(0, 4)),
+            min_players: faker.random.number({ min: 1, max: 4 }),
+            max_players: faker.random.number({ min: 4, max: 20 }),
+            min_playtime: faker.random.number({ min: 5, max: 30 }),
+            max_playtime: faker.random.number({ min: 30, max: 180 }),
+            min_age: faker.random.number({ min: 5, max: 60 }),
+            mechanics: [faker.commerce.productMaterial()],
+            designers: [faker.commerce.department()],
+            artists: [faker.name.findName()],
+            publisher: faker.company.companyName(),
+            family: faker.commerce.productAdjective(),
+            categories: [String(databaseEntries.categories[Math.floor(databaseEntries.categories.length * Math.random())]._id)],
+        }
+    }))
+
+    await submitDocuments('reviews', Review, Array.from({ length: 500 }).map((e, i) => {
+        return {
+            title: faker.lorem.sentence(),
+            content: faker.lorem.paragraph(),
+            game: getRandomElement(databaseEntries.games)._id,
+            author: getRandomElement(databaseEntries.users)._id
+        }
+    }))
+
+    await submitDocuments('likes', Like, Array.from({ length: 2000 }).map((e, i) => {
+        return {
+            game: getRandomElement(databaseEntries.games)._id,
+            user: getRandomElement(databaseEntries.users)._id
```

Comment:
We need to ensure that the user has not liked the same game twice. Could we possibly add this check into the route?

Reply:
This would probably be better to do in the route I think.